Makefile.am

```diff
@@ -0,0 +1,3 @@
+SUBDIRS=src
+ACLOCAL_AMFLAGS=-I m4
+EXTRA_DIST=autogen.sh
```

Comment:
This needs to change to `EXTRA_DIST=bootstrap` ... unless we ditch `bootstrap` altogether and just use `autoreconf -i`.